### PR TITLE
Add DateTime Type

### PR DESCRIPTION
**Pull request by SriramPatibanda**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
- Add Date mock
- Add graphql-iso-date package for date time

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: api/graphql/server.js [hunk 1]
```diff
@@ -17,7 +18,7 @@ async function startApolloServer(typeDefs, resolvers) {
       // TODO: Add auth middleware
     },
     csrfPrevention: true,
-    mocks: true,
+    mocks: {DateTime: () => new Date()},
```
Comment (abhibhaw) [inline api/graphql/server.js:21]: Add a sample link and image mock as well!!

**Review by abhibhaw — CHANGES_REQUESTED** [review]

File: api/graphql/server.js (hunk 1 above)
Comment (abhibhaw) [reply api/graphql/server.js:21]: Reference: [Link](https://www.apollographql.com/docs/apollo-server/testing/mocking/#customizing-mocks)

Comment (SriramPatibanda) [reply api/graphql/server.js:21]: @abhibhaw 
do you mean I should create an image and link type or do you want me to add a sample link and image for location and image of Event type respectively

Comment (abhibhaw) [reply api/graphql/server.js:21]: Leave it!!

**Review by abhibhaw — APPROVED** [review]

**Pull request merged**